src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/index.jsx

```diff
@@ -215,9 +213,9 @@ class GlobalSelectionHeader extends React.Component {
     this.updateStoreIfChange(prevProps, this.props);
   }
 
-  hasMultipleProjectSelection = () => {
-    return new Set(this.props.organization.features).has('global-views');
-  };
+  get hasMultipleProjectSelection() {
+    return this.props.organization.features.includes('global-views');
```

Comment:
I think always dealing with features as sets would help consistency throughout the app

Reply:
IMO it's less boiler plate since it comes out of the API as a list.